src/Form/QuestionType/QuestionTypeAssignee.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form\QuestionType;
+
+use Override;
+use Session;
+
+final class QuestionTypeAssignee extends AbstractQuestionTypeActors
+{
+    #[Override]
+    public function getName(): string
+    {
+        return _n('Assignee', 'Assignees', Session::getPluralNumber());
+    }
+
+    #[Override]
+    public function getWeight(): int
+    {
+        return 30;
+    }
+
+    #[Override]
+    public function getAllowedActorTypes(): array
+    {
+        return ['User', 'Group', 'Supplier'];
```

Comment:
Please use the following notation, it helps IDE / code analyser to link classes. It may be usefull, for instance, if one day we decide to rename them or move them in a namespace.

You will have to add the corresponding `use` statements.

```suggestion
        return [User::class, Group::class, Supplier::class];
```